src/views/Generator/GeneratorTabs/GeneratorTabs.tsx

```diff
@@ -91,6 +91,7 @@ export function GeneratorTabs({
           value="script"
           css={css`
             flex-grow: 1;
+            min-height: 0;
```

Comment:
All this flex juggling is getting very annoying. Perhaps, we need a debt-ticket to simplify the layout implementation.